templates.dist/fields.php

```diff
@@ -214,7 +219,8 @@ function renderFormField(string $fieldName, onOffice\WPlugin\Form $pForm, bool $
 						. $value . ' placeholder="' . esc_attr($rangeDescription) . '">';
 				}
 			} elseif ($typeCurrentInput === FieldTypes::FIELD_TYPE_DATATYPE_TINYINT) {
-				$output = '<fieldset>
+				$additionHidden = $isHiddenField === true ? 'class="oo-hidden-field" disabled ' : '';
+				$output = '<fieldset '.$additionHidden.'>
```

Comment:
Please remove `$additionHidden`, you can use `$hiddenAttribute` instead which is defined at line 126.